### ISSUE TITLE
Date picker from galaxy tool inputs

### ISSUE THREAD

**Issue by annefou** [NordicESMhub/galaxy-tools#22]
The goal is to be able to select a date by displaying a calendar (see https://jqueryui.com/datepicker/).

**Comment by bgruening** [comment]
Galaxy related discussion here: https://github.com/galaxyproject/galaxy/issues/6806

**Comment by bgruening** [comment]
@annefou I guess it would help if you start collecting use-cases. E.g. I imagine you need to select year-rages or even centuries?

**Comment by annefou** [comment]
Yes, year ranges, month ranges  (for climatologies), century ranges, seasons, etc. And you are right, the best to clarify our needs is to collect use-cases. I think we will get there (slowly) together with the developments of climate tools associated to a set of use cases (workflow).

**Comment by annefou** [comment]
Should be simpler using cftime (https://unidata.github.io/cftime/).
